### DISCUSSION TITLE
How do I keep ReVanced up to date?

### DISCUSSION THREAD

**Discussion by spindleclown** [ReVanced/revanced-discussions#569] [Question] [answered]
Do I have to rebuild it every time a new update is released? Or does it update somehow? I apologize for the amateur question.

**Comment by oSumAtrIX** [comment, marked as answer]
You will have to patch each update.

**Reply by spindleclown** [reply]
No but, like, how do I do that? Is there a tutorial? I had to follow what was basically a "For Dummies" tutorial to build RV in the first place. I'm not massively tech savvy.

**Reply by ziphyfyi** [reply]
I think best bet is to just wait for it to officially be released.
